src/EditorFeatures/CSharpTest/SignatureHelp/InvocationExpressionSignatureHelpProviderTests.cs

```diff
@@ -205,6 +205,28 @@ void Goo(int a, int b)
             await TestAsync(markup, expectedOrderedItems);
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.SignatureHelp)]
+        [WorkItem(26713, "https://github.com/dotnet/roslyn/issues/26713")]
+        public async Task TestDelegateParameterWithXmlComment()
+        {
+            var markup = @"
+class C
+{
+    /// <param name=""a"">Parameter docs</param>
+    delegate void SomeDelegate(int a);
+
+    void M(SomeDelegate theDelegate)
+    {
+        [|theDelegate($$|]);
```

Comment:
❗️ I would expect to see tests for the following forms as well:

* `theDelegate.Invoke($$);`
* `theDelegate.BeginInvoke($$);`
* `theDelegate.BeginInvoke(0, $$);` &larr; ensures that the implicit `AsyncCallback` parameter doesn't cause a problem
* `theDelegate.EndInvoke($$);` &larr; ensures that the `IAsyncResult` parameter is handled correctly, and specifically covers the case described separately by @CyrusNajmabadi

Reply:
In the revised PR, EndInvoke no longer receives "special treatment" (only Invoke/BeginInvoke do, see comment), it is treated like any other method (w.r.t. Signature Help/parameter documentation). Should I still add a unit test for it?